docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -583,8 +583,15 @@ Migrating existing tables
 -------------------------
 
 The connector can read from or write to Hive tables that have been migrated to Iceberg.
-There is no Trino support for migrating Hive tables to Iceberg, so you need to either use
-the Iceberg API or Apache Spark.
+A SQL procedure ``system.migrate`` allows the caller to replace
+a Hive table with an Iceberg table, loaded with the source’s data files.
+Table schema, partitioning, properties, and location will be copied from the source table.
```

Comment:
What do you mean?

Reply:
What kind of table properties are copied?

Hive's TBLPROPERTIES are often used to fine-tune table storage behavior, but such are not applicable to Iceberg, and should probably not be copied?

Reply:
All table properties except for `path`, `transient_lastDdlTime`, `serialization.format`. 

I can change the current logic, but it will lose the compatibility with Spark implementation.

Reply:
It's weird for me to copy these, 

> but it will lose the compatibility with Spark implementation.

is it code-documented that we're doing this for compatibility reasons?

Reply:
Left a below comment. 
> // Copy all Hive table properties so that we can undo the migration easily. This is same as Spark implementation.

Reply:
thanks.

btw i thought we create backup table for the purpose of being able to undo